### PR TITLE
Change Var Name

### PR DESCRIPTION
**Pull request by laasso**
Changed 'async' to 'async_req' in some version async is a python reserved word and is not being handled correctly as a method argument.

Thanks!

### CODE REVIEW

File: strava_api_v3/api/athletes_api.py [hunk 1]
```diff
@@ -38,17 +38,17 @@ def get_logged_in_athlete(self, **kwargs):  # noqa: E501
 
         Returns the currently authenticated athlete.  # noqa: E501
         This method makes a synchronous HTTP request by default. To make an
-        asynchronous HTTP request, please pass async=True
```
Comment (sshevlyagin) [inline strava_api_v3/api/athletes_api.py:41]: Looks like the rpelace all was a bit aggressive. I think you mean this line to be 
`asynchronous HTTP request, please pass async_req=True` and there are similar errors in the rest of the file.

**Review by sshevlyagin — COMMENTED** [review]
Thanks! Looks like there's a few typos

File: strava_api_v3/api/athletes_api.py (hunk 1 above)
Comment (laasso) [reply strava_api_v3/api/athletes_api.py:41]: I don't undersant what you mean with that, could you indicate me what i should change? 

It's my first package request, thanks!!!!!!

**Comment by laasso** [comment]
Errors solved from commit #1 [](https://github.com/sshevlyagin/strava-api-v3.1/pull/1/commits/70dbb790a318c2f5b699728bb92b37bb895babf8)

Able to merge, working correctly and comments on point.

@sshevlyagin

File: strava_api_v3/api/gears_api.py [hunk 2]
```diff
@@ -38,18 +38,18 @@ def get_gear_by_id(self, id, **kwargs):  # noqa: E501
...
+        asynchronous HTTP request, please pass async_req=True
+        >>> thread = api.get_gear_by_id(id, async_req=True)
         >>> result = thread.get()
 
-        :param async bool
+        :param async_req bool
         :param int id: The identifier of the gear. (required)
         :return: DetailedGear
-                 If the method is called asynchronously,
+                 If the method is called async_reqhronously,
```
Comment (sshevlyagin) [inline strava_api_v3/api/gears_api.py]: typo, shouldn't have been changed

File: strava_api_v3/api/gears_api.py [hunk 3]
```diff
@@ -60,19 +60,19 @@ def get_gear_by_id_with_http_info(self, id, **kwargs):  # noqa: E501
...
+        asynchronous HTTP request, please pass async_req=True
+        >>> thread = api.get_gear_by_id_with_http_info(id, async_req=True)
         >>> result = thread.get()
 
-        :param async bool
+        :param async_req bool
         :param int id: The identifier of the gear. (required)
         :return: DetailedGear
-                 If the method is called asynchronously,
+                 If the method is called async_reqhronously,
```
Comment (sshevlyagin) [inline strava_api_v3/api/gears_api.py]: typo

File: strava_api_v3/api/streams_api.py [hunk 4]
```diff
@@ -38,20 +38,20 @@ def get_activity_streams(self, id, keys, key_by_type, **kwargs):  # noqa: E501
...
         >>> result = thread.get()
 
-        :param async bool
+        :param async_req bool
         :param int id: The identifier of the activity. (required)
         :param list[str] keys: Desired stream types. (required)
         :param bool key_by_type: Must be true. (required)
         :return: StreamSet
-                 If the method is called asynchronously,
+                 If the method is called async_reqhronously,
```
Comment (sshevlyagin) [inline strava_api_v3/api/streams_api.py]: typo

File: strava_api_v3/api/streams_api.py [hunk 5]
```diff
@@ -62,21 +62,21 @@ def get_activity_streams_with_http_info(self, id, keys, key_by_type, **kwargs):
...
```
Comment (sshevlyagin) [inline strava_api_v3/api/streams_api.py]: typo

File: strava_api_v3/api/streams_api.py [hunk 6]
```diff
@@ -153,20 +153,20 @@ def get_segment_effort_streams(self, id, keys, key_by_type, **kwargs):  # noqa:
...
         >>> result = thread.get()
 
-        :param async bool
+        :param async_req bool
         :param int id: The identifier of the segment effort. (required)
         :param list[str] keys: The types of streams to return. (required)
         :param bool key_by_type: Must be true. (required)
         :return: StreamSet
-                 If the method is called asynchronously,
+                 If the method is called async_reqhronously,
```
Comment (sshevlyagin) [inline strava_api_v3/api/streams_api.py]: typo

**Review by sshevlyagin — COMMENTED** [review]
Please fix all of the `async_reqhronously` typos.

**Comment by laasso** [comment]
All `async_reqhronously` typos solved.

Please let me know if anything else isn't OK.

**Review by sshevlyagin — APPROVED** [review]

**Pull request merged**